File: solutions/q1768/q1768_solution.cs
public class Solution {
    public string MergeAlternately(string word1, string word2) {
        StringBuilder word3 = new StringBuilder();
        int length1 = word1.Length;
        int length2 = word2.Length;
        int maxLength = Math.Max(length1, length2);

        for (int i = 0; i < maxLength; i++) {
            if (i < length1)
                word3.Append(word1[i]);
            if (i < length2)
                word3.Append(word2[i]);
        }

        return word3.ToString();
    }
}

File: solutions/q1768/q1768_solution.kt
class Solution {
    fun mergeAlternately(word1: String, word2: String): String {
        return buildString {
            val minLength = minOf(word1.length, word2.length)

            for (i in 0 until minLength) {
                append(word1[i])
                append(word2[i])
            }

            append(word1.substring(minLength))
            append(word2.substring(minLength))
        }
    }
}
